From b0f859da4782667ddb6d7f36a1ca041fddcf1101 Mon Sep 17 00:00:00 2001
From: Mykola Lysenko <mykola.lysenko@amd.com>
Date: Thu, 2 Jul 2015 11:59:36 -0400
Subject: [PATCH 330/343] drm/amd/amdgpu: [DM] Add blanking before modeset

In case we have view change only, still need to blank
because display clock can change

Signed-off-by: Mykola Lysenko <mykola.lysenko@amd.com>
Reviewed-by: Eagle Yeh <eagle.yeh@amd.com>
Signed-off-by: Sanjay R Mehta <sanju.mehta@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_dm_types.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_dm_types.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_dm_types.c
index 2dd2341..d4b5093 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_dm_types.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_dm_types.c
@@ -536,6 +536,7 @@ bool amdgpu_dm_mode_set(
 	}
 
 
+	dal_set_blanking(adev->dm.dal, acrtc->crtc_id, true);
 	/* the actual mode set call */
 	ret = dal_set_path_mode(adev->dm.dal, pms);
 
-- 
1.9.1

